server/database/models/ModelDocumentRepository.js

```diff
@@ -0,0 +1,78 @@
+const AbstractRepository = require("./AbstractRepository");
+
+class ModelDocumentRepository extends AbstractRepository {
+  constructor() {
+    // Call the constructor of the parent class (AbstractRepository)
+    // and pass the table name "Model_Docs" as configuration
+    super({ table: "Model_Docs" });
+  }
+
+  // The C of CRUD - Create operation
+  async create(model) {
+    // Execute the SQL INSERT query to add a new model to the "Model_docs" table
+    const [result] = await this.database.query(
+      `insert into ${this.table} (Type_Doc,Languages_source,Status,Real_Path_Emplacement,Id_Client,Id_Translator) values (?, ?, ?, ?, ?, ?)`,
+      [
+        model.TypeDocument,
+        model.LanguageSource,
+        model.Status,
+        model.PathUploadedFile,
+        model.IdClient,
+        model.idTranslator,
+      ]
+    );
+
+    // Return the ID of the newly inserted client
+    return result.insertId;
+  }
+
+  // The Rs of CRUD - Read operations
+
+  async read(id) {
+    // Execute the SQL SELECT query to retrieve a specific model document by its ID
+    const [rows] = await this.database.query(
+      `select * from ${this.table} where Id_Doc = ?`,
```

Comment:
table names are in snake case and the name id is sufficient to name the primary key field